src/main/java/Ui.java

```diff
@@ -0,0 +1,103 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+    static String indentation = "    ";
+    static String logo = " ____        _\n"
+            + "|  _ \\ _   _| | _____\n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    static String prompt = "What can I do for you?";
+    static String demarcation = Ui.indentation + "-------------------------------------------------------------------";
```

Comment:
Could this attribute be made private static final so that it is a constant variable? The naming can then be done in full uppercase with underscores.